vdsm.spec.in

```diff
@@ -215,12 +210,8 @@ Requires: python3-policycoreutils
 
 # qemu-kvm
 
-%if 0%{?centos}
-Requires: qemu-kvm >= 15:6.0.0-28
```

Comment:
Is dropping the epoch (15) from version requirement safe here? Or would this essentially mean "install any recent version" because missing epoch equals zero?

Reply:
I think the epoch should be there.  Fixed, thanks for noticing this.